use-cases/inferencing/serving-with-vllm/README.md

```diff
@@ -0,0 +1,513 @@
+  INFO:     Waiting for application startup.
+  INFO:     Application startup complete.
+  INFO:     Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)
+  ```
+
+## Serve the deployed model through curl and a web chat interface
+
+- Test your deployed model through the CLI
+
+  ```sh
+  kubectl port-forward svc/vllm-openai -n ${NAMESPACE} 8000
+  ```
+
+  Run the curl prompt with your values
+
+  ```sh
+  USER_PROMPT="I'm looking for comfortable cycling shorts for women, what are some good options?"
+  MODEL_ID=""
+
+  curl http://localhost:8000/v1/chat/completions \
+    -H "Content-Type: application/json" \
+    -d '{
+        "model": "${MODEL_ID}",
+        "messages": [
+            {"role": "user", "content": "${USER_PROMPT}"}],
+             "temperature": 0.70,
+             "top_p": 1.0,
+             "top_k": 1.0,
+             "max_tokens": 256
+    }'  
+  ```
+
+- You can also deploy a gradio chat interface to view the model chat interface. [OPTIONAL]
+  
+  ```sh
+  MODEL-ID=<your-model-id>
+  sed -i -e "s|_MODEL-ID_|${MODEL-ID}|" manifests/gradio.yaml
+  ```
+
+  ```sh
+  sed -i -e "s|_NAMESPACE_|${NAMESPACE}|" manifests/gradio.yaml
+  kubectl apply -f manifests/gradio.yaml
+  ```
+
+### Production Metrics
+
+vLLM exposes a number of metrics that can be used to monitor the health of the system. These metrics are exposed via the `/metrics` endpoint on the vLLM OpenAI compatible API server.
+
+  ```sh
+  curl http://localhost:8000/metrics
+  ```
+
+### View vLLM serving metrics for your model on GKE
+
+You can configure monitoring of the metrics above using the [pod monitoring](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#gmp-pod-monitoring)
+
+  ```sh
+  kubectl apply -f manifests/pod_monitoring.yaml
+  ```
+
+### Create a dashboard for Cloud Monitoring to view vLLM metrics
+
+Cloud Monitoring provides an [importer](https://cloud.google.com/monitoring/dashboards/import-grafana-dashboards) that you can use to import dashboard files in the Grafana JSON format into Cloud Monitoring
+
+ Clone github repository
+
+  ```sh
+  git clone https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples
+  ```
+
+ Change to the directory for the dashboard importer:
+
+  ```sh
+  cd monitoring-dashboard-samples/scripts/dashboard-importer
+  ```
+
+The dashboard importer includes the following scripts:
+
+- import.sh, which converts dashboards and optionally uploads the converted dashboards to Cloud Monitoring.
+- upload.sh, which uploads the converted dashboards—or any Monitoring dashboards—to Cloud Monitoring. The import.sh script calls this script to do the upload.
+
+ Import the dashboard
+
+  ```sh
+  ./import.sh ./configs/grafana.json ${PROJECT_ID}
+  ```
+
+  When you use the import.sh script, you must specify the location of the Grafana dashboards to convert. The importer creates a directory that contains the converted dashboards and other information.
+
+
+### Run Batch inference on GKE
```

Comment:
would we want to make a new MD file for this path

Reply:
Or do we just want to include the already converted dashboard?